python/python/glide/constants.py

```diff
@@ -18,6 +18,7 @@
     int,
     None,
     Dict[str, T],
+    Mapping[str, "TResult"],
```

Comment:
Is it possible?
```suggestion
    Mapping[str, TResult],
```

Reply:
Nope 😕 , this is the syntax of what you're suggesting